ufw/config/file.sls

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- set sls_package_install = tplroot ~ '.package.install' %}
+{%- from tplroot ~ "/map.jinja" import ufw with context %}
+{%- from tplroot ~ "/libtofs.jinja" import files_switch with context %}
+
+include:
+  - {{ sls_package_install }}
+
+ufw-default-file-file-managed:
+  file.managed:
+    - name:     {{ ufw.default_file }}
+    - user:     root
+    - group:    root
+    - template: jinja
+    - source: {{ files_switch(['ufw.default.tmpl', 'ufw.default.tmpl.jinja'],
+                              lookup='ufw-default-file-file-managed'
+                 )
+              }}
+    - require:
+      - sls: {{ sls_package_install }}
+    - context:
+        ufw_settings: {{ ufw.settings | json }}
+
+ufw-sysctl-file-file-managed:
+  file.managed:
+    - name:     {{ ufw.sysctl_file }}
+    - user:     root
+    - group:    root
+    - template: jinja
+    - source: {{ files_switch(['ufw.sysctl.tmpl', 'ufw.sysctl.tmpl.jinja'],
+                              lookup='ufw-sysctl-file-file-managed'
+                 )
+              }}
+    - require:
+      - sls: {{ sls_package_install }}
+    - context:
+        ufw_ssysctl: {{ ufw.sysctl | json }}
```

Comment:
Did you mean `ufw_ssysctl` or `ufw_sysctl`?  If you wanted the double `ss`, then please ignore this.